dns/zone-configs/k8s.io._0_base.yaml

```diff
@@ -29,9 +31,10 @@ dummy:
 # Our vanity redirector.  This is not just 'k8s.io', on the off chance this
 # becomes different from the main record. (@thockin)
 redirect:
-  type: A
-  value: 34.107.204.206
-
+  - type: A
```

Comment:
I guess the DNS tests will just work, but I guess we'll find out?

Reply:
this is the part that scaries me the most, since everything depends on redirect and is moving to an array :crossed_fingers: 
I saw similar records before, but :shrug: